src/main/java/org/bitsofinfo/hazelcast/spi/docker/swarm/dnsrr/discovery/DockerDNSRRDiscoveryStrategy.java

```diff
@@ -68,10 +68,10 @@ public DockerDNSRRDiscoveryStrategy(
             return discoveryNodes;
         }
 
-        Set<InetAddress> serviceNameResolutions =
-                new HashSet<>();
+        // TODO robin - tighten scope of these variables
```

Comment:
remove this

Reply:
Already done in a pending PR which tightens variable scopes.